optuna_integration/wandb/wandb.py

```diff
@@ -37,7 +37,7 @@ class WeightsAndBiasesCallback:
     .. note::
         Users who want to run multiple Optuna studies within the same process
         should call ``wandb.finish()`` between subsequent calls to
-        ``study.optimize()``. Calling ``wandb.finish()`` is not necessary
+        :meth:`~optuna.study.Study.optimize`. Calling ``wandb.finish()`` is not necessary
```

Comment:
Currently, only `optimize` is shown, which is a bit vague in the text.
```suggestion
        :meth:`optuna.study.Study.optimize`. Calling ``wandb.finish()`` is not necessary
```